### DISCUSSION TITLE
registerBiometricSecret. The operation couldn’t be completed. (CordovaPlugins.KeychainError error 1.)

### DISCUSSION THREAD

**Discussion by lenarmazitov** [NiklasMerz/cordova-plugin-fingerprint-aio#399] [General]
IOS Simulator (latest XCode)

When I'm trying to register secret through **FingerprintAIO.registerBiometricSecret**, I get error

```
{code: -100, message: "The operation couldn’t be completed. (CordovaPlugins.KeychainError error 1.)"}
```

I tried to enable Keychain sharing and add Keychain group, but it did not helps.

"@capacitor/core": "3.3.4",
"@ionic-native/core": "^5.36.0",
"@ionic-native/fingerprint-aio": "^5.36.0",
"cordova-plugin-fingerprint-aio": "^5.0.1",

**Comment by LilyBeasty** [comment]
Having the same issue, both for fingerprint and faceID iOS devices. Anyone have any ideas?

**Comment by giloser** [comment]
Hi, recently got this issue on some phone any solution?
